src/FakeItEasy/Generated/CodeGen/CodeGen.ArgumentValidationConfigurationExtensionsGenerator/ArgumentValidationConfigurationExtensions.StronglyTyped.cs

```diff
@@ -249,6 +248,5 @@ public static partial class ArgumentValidationConfigurationExtensions
                 return argumentsPredicate(args.Get<T1>(0), args.Get<T2>(1), args.Get<T3>(2), args.Get<T4>(3), args.Get<T5>(4), args.Get<T6>(5), args.Get<T7>(6), args.Get<T8>(7));
             });
         }
-
     }
-}
+}
```

Comment:
I'm surprised there's no newline here. Will investigate!

Reply:
Turns out my git diff displayer (delta) has output that _I_ think looks like there used to be no newline at the end of file, and I added one. But it was opposite. Restoring newline in a fixup.